velox/dwio/common/FormatData.h

```diff
@@ -86,7 +86,7 @@ class FormatData {
   /// is in FormatData the provider is at end. For ORC/DWRF the type
   /// dependent stream positions are accessed via the provider. The
   /// provider is valid until next call of this.
-  virtual dwio::common::PositionProvider seekToRowGroup(uint32_t index) = 0;
+  virtual dwio::common::PositionProvider seekToRowGroup(int64_t index) = 0;
```

Comment:
why uint64_t? Thanks!

Reply:
Usually we prefer signed type for numeric values because it won't create surprise when compare/subtract with negative numbers.  See https://godbolt.org/z/n4zxo15bn